src/shared/bus-wait-for-jobs.c

```diff
@@ -279,8 +279,11 @@ static int check_wait_response(BusWaitForJobs *d, WaitJobsFlags flags, const cha
                 return -EOPNOTSUPP;
         else if (streq(d->result, "once"))
                 return -ESTALE;
-        else if (STR_IN_SET(d->result, "done", "skipped"))
+        else if (STR_IN_SET(d->result, "done", "skipped")) {
+                if (flags & BUS_WAIT_JOBS_LOG_SUCCESS)
+                        log_info("Unit %s started.", strna(d->name));
                 return 0;
+        }
```

Comment:
This should mention the job succeeded, since this function logs about jobs, not units.